packages/page-supersig/src/augment-supersig.ts

```diff
@@ -0,0 +1,26 @@
+import '@polkadot/rpc-core/types/jsonrpc';

```

Comment:
We cannot bring in additional augmentation files - we only use the defaults from the JS API.

Reply:
this can be removed will remove.